OrbitSshQt/include/OrbitSshQt/StateMachineHelper.h

```diff
@@ -0,0 +1,228 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_SSH_QT_BASE_H_
+#define ORBIT_SSH_QT_BASE_H_
+
+#include <QObject>
+
+#include "OrbitBase/Logging.h"
+#include "OrbitSsh/Error.h"
+#include "OrbitSshQt/Error.h"
+
+namespace OrbitSshQt {
+
+/**
```

Comment:
javadoc? :)

we should probably agree on style /* */ or //

Reply:
I just liked the asterisks. ;-) Went with /* */ now. Thanks.